src/components/views/settings/Notifications.tsx

```diff
@@ -470,6 +473,18 @@ export default class Notifications extends React.PureComponent<IProps, IState> {
         });
     };
 
+    private validateKeyword = withValidation({
+        description: () => undefined,
+        hideDescriptionIfValid: true,
+        rules: [
+            {
+                key: "hasOnlySpace",
+                test: ({ value }) => !value || value.trim().length !== 0,
+                invalid: () => _t("Keywords cannot contain only space ' '"),
```

Comment:
after looking at the spec, I'm not seeing a reason why this rule would be in place. What issues does it cause to have a space? do we need to update the spec?